docs/Documentation/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md

```diff
@@ -144,84 +144,97 @@ The execution result of this SQL statement is as follows:
 <center><img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/13203019/51577450-dcfe0800-1ef4-11e9-9399-4ba2b2b7fb73.jpg"></center>
 
 ### Down-Frequency Aggregate Query
-This section mainly introduces the related examples of down-frequency aggregation query, using the [GROUP BY clause](/#/Documents/progress/chap5/sec4), which is used to partition the result set according to the user's given partitioning conditions and aggregate the partitioned result set. IoTDB supports partitioning result sets according to time intervals, and by default results are sorted by time in ascending order. You can also use the [Java JDBC](/#/Documents/progress/chap4/sec2) standard interface to execute related queries.
+This section mainly introduces the related examples of down-frequency aggregation query, 
+using the [GROUP BY clause](/#/Documents/progress/chap5/sec4), 
+which is used to partition the result set according to the user's given partitioning conditions and aggregate the partitioned result set. 
+IoTDB supports partitioning result sets according to time interval and customized sliding step whose default value is equal to time interval and must be larger than time interval, and by default results are sorted by time in ascending order. 
```

Comment:
`whose default value is equal to time interval and must be larger than time interval`
->
which should not be smaller than the time interval and defaults to equal the time interval if not set.